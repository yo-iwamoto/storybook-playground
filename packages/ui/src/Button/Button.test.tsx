import { render } from '@testing-library/react';
import { Button } from '.';

describe('Button', () => {
  it('正しく表示されること', () => {
    const { getByRole, asFragment } = render(
      <Button type="button">送信</Button>,
    );

    expect(getByRole('button', { name: '送信' })).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
