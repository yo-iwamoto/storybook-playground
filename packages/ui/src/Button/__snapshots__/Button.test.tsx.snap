// Vitest Snapshot v1

exports[`Button > 正しく表示されること 1`] = `
<DocumentFragment>
  <button
    type="button"
  >
    送信
  </button>
</DocumentFragment>
`;
